manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -377,6 +377,8 @@ defaults:
     proxy_init:
       # Base name for the proxy_init container, used to configure iptables.
       image: proxyv2
+      # Allows override if detection incorrectly assumes rules are already applied. Consider removing after several releases with no reported issues.
```

Comment:
```suggestion
      # Bypasses iptables reconciliation, and attempts to apply iptables rules regardless of table state, which may cause unrecoverable failures. Do not use. This flag will be removed in future releases.
```

Reply:
Done, I changed your proposal a bit because the flag bypasses iptables idempotency handling and not the reconciliation :)

https://github.com/istio/istio/blob/673776d68667b01a36016a678df3a609f3d14432/manifests/charts/istio-control/istio-discovery/values.yaml#L380-L381

Reply:
yep, as long as it's sufficiently clear/scary.